src/components/presentations/Profilepage/__test__/Profilepage.actions.test.js

```diff
@@ -1,9 +1,12 @@
+/* eslint-disable no-dupe-keys */
```

Comment:
@sojida  any reason for disabling this rule?

Reply:
I'll take it off. That's an oversight from my end.